qemu/qemu.go

```diff
@@ -384,24 +385,25 @@ func (n NetDeviceType) QemuNetdevParam() string {
 	}
 }
 
+var vfio = "vfio-pci"
+
 // QemuDeviceParam converts to the QEMU -device parameter notation
 func (n NetDeviceType) QemuDeviceParam() string {
 	switch n {
 	case TAP:
-		return "virtio-net-pci"
+		return string(VirtioNet)
```

Comment:
why VirtioNet? this should be VirtioNetPCI

Reply:
@devimc we know this gets aliased correctly inside qemu to -pci or -ccw based on platform so this removes another thing that needs to be conditionally controlled.